os/arch/arm/src/armv7-a/arm_assert.c

```diff
@@ -607,14 +607,18 @@ void up_assert(const uint8_t *filename, int lineno)
 	 */
 	if (!IS_SECURE_STATE()) {
 		print_assert_detail(filename, lineno, fault_tcb, asserted_location);
+	} else {
+		if (mm_check_heap_corruption(g_kmmheap) != OK) {
```

Comment:
If we call mm_check_heap_corruption() here, then kernel heap details might get printed even if SECURE_STATE is present. Is this ok?

Reply:
Thank you for review.

I added checking secure_state in mfdbg and heap_dbg.
https://github.com/Samsung/TizenRT/commit/0f974de4254efaa7b986a3a6b7f9659c3a2b065e

So It is not printed.